.github/workflows/pr-test-e2e.yml

```diff
@@ -85,6 +84,10 @@ jobs:
           cd test/e2e/ginkgo
           KFLEX_DISABLE_CHATTY=true ginkgo -vv --trace --no-color --fail-fast $TEST_FLAGS -- -kubestellar-setup-flags="--kubestellar-controller-manager-verbosity 5 --transport-controller-verbosity 5"
 
+      - name: Dump kubeflex-controller-manager logs
+        if: always()
+        run: kubectl logs -n kubeflex-system deploy/kubeflex-controller-manager
```

Comment:
(A) I would move this down later, see how there is something of a progression of dependency in the debug logging?

(B) It is helpful to also dump the log from the previous run of the same container (`-p`) if that's available.